File: src/main/java/net/veloxia/logis/exception/LanguageNotFoundException.java
package net.veloxia.logis.exception;

/**
 * Exception thrown when a language JSON file cannot be found at the given path.
 *
 * @author xRookieFight
 * @since 23/03/2026
 */
public class LanguageNotFoundException extends RuntimeException {

    public LanguageNotFoundException(String filePath) {
        super("Language not found in path: " + filePath);
    }
}

File: src/main/kotlin/net/veloxia/logis/exception/LanguageNotFoundException.kt
package net.veloxia.logis.exception

/**
 * Exception thrown when a language JSON file cannot be found at the given path.
 *
 * @param filePath The path to the missing language file
 * @author xRookieFight
 */
class LanguageNotFoundException(
    filePath: String
) : Exception("Language not found in path: $filePath")
